test/server/cards/01_SOR/events/IHadNoChoice.spec.ts

```diff
@@ -0,0 +1,92 @@
+describe('I Had No Choice', function() {
+    integration(function(contextRef) {
+        describe('I Had No Choice\'s ability', function() {
+            describe('when there are two or more non-leader units in play', function() {
+                beforeEach(function () {
+                    contextRef.setupTest({
+                        phase: 'action',
+                        player1: {
+                            hand: ['i-had-no-choice'],
+                            deck: ['foundling', 'pyke-sentinel', 'atst', 'cartel-spacer', 'battlefield-marine'],
+                            groundArena: [{ card: 'wampa', upgrades: ['experience', 'resilient'] }, 'warzone-lieutenant'],
```

Comment:
What's the reason for the experience on the wampa?

Reply:
Just to ensure that token upgrades are destroyed, `MoveCardSystem` didn't support that previously.

Reply:
Makes sense. Trying to figure out if there's an actual check we could do that the behavior is correct. Or was it just throwing an error previously?

Reply:
It was throwing an error saying something like "you cannot move experience tokens to discard pile".